pcdsdevices/ladm.py

```diff
@@ -0,0 +1,423 @@
+        gamma1 = y1ToGamma(self.y1.wm())
+        gamma2 = y1ToGamma(self.y2.wm())
+        gamma = gamma2-gamma1
+        message = f"gamma = {gamma}(y1 = {self.y1.wm()}/y2 = {self.y2.wm()})"
+        print(message)
+        return gamma, message
+
+    def mvrGamma(self, theta, wait=False):
+        y1_var, y2_var, dy = GammaToMotors(theta)
+        # gamma1 = y1ToGamma(self.y1.wm())
+        # gamma2 = y1ToGamma(self.y2.wm())
+        self.y1.mvr(y1_var, wait=wait)
+        self.y2.mvr(y2_var, wait=wait)
+
+    def setTheta(self, value):
+        """ set x1, x2 and  z for Theta value """
+        x1_var, x2_var, z_var = ThetaToMotors(value)
+        self.x1.set(x1_var)
+        self.x2.set(x2_var)
+        self.z.set(z_var)
+
+    def moveX(self, x):
+        """
+        whole ladm move horizontally and keep same Z distance from diff
+        """
+        if ((x <= self.__lowlimX) or (x >= self.__hilimX)):
+            logger.debug(f"""Asked to move {self.XT.name}
+                           outside limit, aborting""")
+        else:
+            try:
+                x1_var = xTox12(x)
+                x2_var = xTox12(x)
+                z_var = xToz(x)
+                z_now = self.z.wm()
+                if z_var > z_now:
+                    print(f"Moving z to o {z_var}")
+                    self.z.mv(z_var)
+                    self.z.wait()
+                    print(f"Moving x1 to {x1_var} and x2 to {x2_var}")
+                    self.x1.mv(x1_var)
+                    self.x2.mv(x2_var)
+                else:
+                    print(f"Moving x1 to {x1_var} and x2 to {x2_var}")
+                    self.x1.mv(x1_var)
+                    self.x2.mv(x2_var)
+                    self.x1.wait()
+                    self.x2.wait()
+                    print(f"Moving z to {z_var}")
+            except KeyboardInterrupt:
+                self.stop()
+
+    def tweakH(self, x):
+        """
+        whole ladm move horizontally and keep same Z distance from diff
+        """
+        try:
+            x1_var = xTox12(x)
+            x2_var = xTox12(x)
+            z_var = xToz(x)
+            z_now = self.z.wm()
+            if z_var > z_now:
+                print(f"Moving z to {z_var}")
+                self.z.mvr(z_var - z_now)
+                self.z.wait()
+                print("Done")
+                print(f"Moving x1 to {x1_var} and x2 to {x2_var}")
+                self.x1.mvr(x1_var)
+                self.x2.mvr(x2_var)
+                print("Done")
+            else:
+                print(f"Moving x1 to {x1_var} and x2 to {x2_var}")
+                self.x1.mvr(x1_var)
+                self.x2.mvr(x2_var)
+                self.x1.wait()
+                self.x2.wait()
+                print("Done")
+                print(f"Moving z to {z_var}")
+                self.z.mvr(z_var - z_now)
+                print("Done")
+        except KeyboardInterrupt:
+            self.stop()
+
+    def mvrV(self, y):
```

Comment:
Enhancement / Improvement idea / Musing: 

It looks like we're most interested in derived quantities (V, H, X, Theta), rather than the motors that have been defined in this class (x1, y1, z, etc)

Should we be wrapping these quantities in pseudo positioners so that we can use normal manipulate them with normal `FltMvInterface` methods?